src/states/node.rs

```diff
@@ -721,7 +720,7 @@ impl Node {
                    self,
                    peer_id,
                    hop_msg);
-            return Err(RoutingError::UnknownConnection(peer_id));
+            *self.name()
```

Comment:
Definitely like to see a code comment here, lest any future code-explorer be hopelessly confused (that's also why I advocated `None`).